src/child_nutrition.py

```diff
@@ -0,0 +1,160 @@
+    '''Loads Excel worksheets into DataFrames
+
+    Arguments:
+    src -- path to an xlsx file
+
+    Returns:
+    A dictionary mapping worksheet names to DataFrames
+    '''
+    table = pd.read_excel(src, sheet_name=None, engine='openpyxl')
+    for k in table:
+        # drop empty rows (i.e. all values are NaN)
+        table[k].dropna(axis=0, how='all', inplace=True)
+
+        # fill NaN as zero
+        table[k].fillna(0, inplace=True)
+
+        # ensure we're indexing from 0
+        table[k].reset_index(inplace=True)
+
+        # drop column of old indices
+        table[k].drop(table[k].columns[0], axis=1, inplace=True)
+
+    return table
+
+
+def rename_columns(table: Dict[str, pd.DataFrame]) -> None:
+    '''In-place renames the columns in each DataFrame to snake_case
+
+    Arguments:
+    table -- A dictionary mapping str keys to dataframes
+    '''
+    for k in table:
+        table[k].columns = [
+            'county',
+            'breakfast_count',
+            'lunch_count',
+            'sfsp_count',
+            'afterschool_meals_count']
+
+
+def add_fips_column(table: Dict[str, pd.DataFrame]) -> None:
+    '''Adds a column in-place to each DataFrame where each row value is
+    the FIPS code for the county named in the 'county' column
+
+    Arguments:
+    table -- A dictionary mapping str keys to DataFrames
+    '''
+    fips = county_fips()
+    # We're going to make a separate dictionary where the keys are just
+    # the county name in all caps. Note that we're trimming out spaces
+    # as well to account for e.g. DE WITT vs. DEWITT, which is written
+    # both ways depending on the data source
+    abbr_fips = dict()
+    for k in fips:
+        abbr_key = k.upper().replace(' COUNTY, ILLINOIS', '').replace(' ', '')
+        abbr_fips[abbr_key] = fips[k]
+
+    # Now that we have the new dictionary, populate the fips column
+    for k in table:
+        table[k]['fips'] = None
+        for index, row in table[k].iterrows():
+            fip = abbr_fips[row[0].replace(' ', '').upper()]
+            table[k].iloc[index, -1] = fip
+
+        # Set index to fips column
+        table[k].set_index('fips', inplace=True)
+
+        # drop column of county names
+        table[k].drop('county', axis=1, inplace=True)
+
+
+def to_dict(table: Dict[str, pd.DataFrame]) -> Dict[str, Dict[str, Dict[str, Any]]]:  # noqa E501
+    '''Combines several dataframes into a single dictionary structure.
+
+    Arguments:
+    table -- A dictionary mapping str keys to DataFrames
+
+    Returns:
+    A dictionary with the structure:
+    <sheet name>: {
+        <fips>: {
+            "breakfast_count": 0,
+            "lunch_count": 0,
+            "sfsp_count": 0,
+            "afterschool_meals_count": 0
+        }
+    }
+    '''
+    output = dict()
+    for k in table:
+        output[k] = table[k].to_dict(orient='index')
+    return output
+
+
+def make_bins(table: Dict[str, pd.DataFrame], bin_func: Callable) -> Dict:
```

Comment:
If you want you can enforce the signature of Callable with Callable[[List, Of, Parameter, Types], ReturnType]

Reply:
Will do.